tests/cupy_tests/core_tests/test_ndarray_cuda_array_interface.py

```diff
@@ -9,18 +9,21 @@ class DummyObjectWithCudaArrayInterface(object):
 
     def __init__(self, a):
         self.a = a
+        self._desc = None
```

Comment:
I think we don't need to cache the attribute as this class is only used for tests.

Reply:
Ah sure, that's true. I made it that this class is an identical copy of the one from `test_from_data.py` except that the `set_mask()` method is removed, but they don't have to be in sync.